domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/validation/ValidationErrorMessageCreator.java

```diff
@@ -83,7 +80,11 @@ String createErrorMessage(EventEntityMissingForPageTypeDisplayGroupError
 
     String createErrorMessage(DisplayGroupInvalidShowConditionError displayGroupInvalidShowConditionError);
 
-    String createErrorMessage(DisplayGroupInvalidShowConditionField displayGroupInvalidShowConditionField);
+    String createErrorMessage(DisplayGroupInvalidShowConditionFieldForEvent displayGroupInvalidShowConditionFieldForEvent);
```

Comment:
perhaps we could just call these new error parameters 'error'?

Reply:
Shall I change it everywhere as I was following the convention in the class?